specs/vep-1243-vdk-meta-jobs/README.md

```diff
@@ -90,6 +110,20 @@ In this context, a component is any separate software process.
 
 -->
 
+![high-level-diagram.png](high-level-diagram.png)
```

Comment:
High-level design png is missing the context of VDK SDK.

Reply:
I am not really sure how does VDK SDK relate to the High-level design?

Reply:
Where does the vdk-meta-jobs plugin instance live? What runtime, what context?